templates/new_gem/lib/fluent/plugin/storage.rb.erb

```diff
@@ -0,0 +1,50 @@
+<%= preamble %>
+
+require "fluent/plugin/storage"
+
+module Fluent
+  module Plugin
+    class <%= class_name %> < Fluent::Plugin::Storage
+      Fluent::Plugin.register_storage("<%= plugin_name %>", self)
+
+      attr_reader :store
+
+      # Set persistent true by default
+      config_set_default :persistent, true
```

Comment:
Is it needed?
Why does it override `Fluent::Plugin::Storage`'s default setting?

Reply:
It should not set by default, it depends on storage plugin.